controller/service.rst

```diff
@@ -110,7 +110,7 @@ which is a common practice when following the `ADR pattern`_
         # config/routes.yaml
         hello:
             path:     /hello/{name}
-            defaults: { _controller: app.hello_controller }
+            controller: app.hello_controller
```

Comment:
Just asking, shouldn't this be instead:

```
controller: App\Controller\HelloController
```

Reply:
It may be as FQCN and in snake_case.

Above in the page it is mentioned - that user can use both ways for referencing (https://github.com/symfony/symfony-docs/blob/5.2/controller/service.rst#referencing-your-service-from-routing).

Do you prefer to change it to FQCN style?

Reply:
I've checked this again. It's fine to keep the snake_case notation in this example. Thanks!